Students/Jacoby/Mechanum/commands/DriveTrainCommand.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.DriveTrain;
+
+/** An example command that uses an example subsystem. */
+public class DriveTrainCommand extends CommandBase 
+{
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+  private final DriveTrain m_subsystem;
+
+
+
+  public DriveTrainCommand(DriveTrain subsystem) {
+    m_subsystem = subsystem;
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(subsystem);
+  }
+
+
+  @Override
+  public void execute() 
+  {
+
```

Comment:
You are missing your subsystem method calls in the execute() method of drive train command. Remember that DriveTrainCommand.execute() is being called constantly on the RIO scheduler. Right now because it is blank, it wont do anything. You would have something like subsystem.drive() in the execute method.

Reply:
Resolved.